test/models/Company/Repository.test.ts

```diff
@@ -131,20 +136,160 @@ describe("CompanyRepository", () => {
     expect(await CompanyRepository.findByUuid(uuid)).not.toBeNull();
     await CompanyRepository.truncate();
     await expect(CompanyRepository.findByUuid(uuid)).rejects.toThrow();
   });
+
+  describe("Update", () => {
+    beforeEach(() => 0);
+
+    it("updates only company attributes", async () => {
+      const companyCompleteData = companyMocks.nineteenCompaniesWithCompleteData()[9];
+      const { uuid } = await CompanyRepository.create(companyCompleteData);
+      const dataToUpdate = {
+        cuit: "30711311773",
+        companyName: "Devartis SA",
+        slogan: "new slogan",
+        description: "new description",
+        logo: "",
+        website: "http://www.new-site-com",
+        email: "old@devartis.com"
+      };
+      const company = await CompanyRepository.update({ uuid, ...dataToUpdate });
+      expect(company).toMatchObject(dataToUpdate);
+    });
+
+    it("updates company phone numbers by adding new", async () => {
+      const companyCompleteData = companyMocks.nineteenCompaniesWithCompleteData()[10];
+      const { uuid } = await CompanyRepository.create(companyCompleteData);
+      const company = await CompanyRepository.update({ uuid, phoneNumbers: ["1159821066"] });
+      const phoneNumbers = (await company.getPhoneNumbers()).map(({ phoneNumber }) => phoneNumber);
+      expect(phoneNumbers).toEqual(["1159821066"]);
+    });
+
+    it("updates company phone numbers by adding new ones and deleting missing ones", async () => {
+      const companyCompleteData = companyMocks.nineteenCompaniesWithCompleteData()[11];
+      const { uuid } = await CompanyRepository.create(
+        {
+          ...companyCompleteData,
+          phoneNumbers: ["1159871234", "1160393692"]
+        }
+      );
+      const newPhoneNumbers = ["1143075222"];
+      const company = await CompanyRepository.update({ uuid, phoneNumbers: newPhoneNumbers });
+      const phoneNumbers = (await company.getPhoneNumbers()).map(({ phoneNumber }) => phoneNumber);
+      expect(phoneNumbers).toHaveLength(1);
+      expect(phoneNumbers).toEqual(newPhoneNumbers);
+    });
+
+    it("updates company phone numbers by adding a new one", async () => {
+      const companyCompleteData = companyMocks.nineteenCompaniesWithCompleteData()[12];
+      const initialPhoneNumbers = ["1159871234", "1160393692"];
+      const { uuid } = await CompanyRepository.create(
+        {
+          ...companyCompleteData,
+          phoneNumbers: initialPhoneNumbers
+        }
+      );
+      const newPhoneNumbers = [...initialPhoneNumbers, "1143075222"];
+      const company = await CompanyRepository.update({ uuid, phoneNumbers: newPhoneNumbers });
+      const phoneNumbers = (await company.getPhoneNumbers()).map(({ phoneNumber }) => phoneNumber);
+      expect(phoneNumbers).toHaveLength(3);
+      expect(phoneNumbers).toEqual(expect.arrayContaining(newPhoneNumbers));
+    });
+
+    it("does not updatea any company phone number", async () => {
```

Comment:
updatea

Reply:
explicaria mejor este caso, veo que es algo como "does not update company phone numbers when they are not given"

esto no es un change request, pero yo a este update lo habria hecho sin tocar cosas como los phone numbers porque es codigo de más para lo que hoy se hace en front